packages/twenty-front/src/modules/views/components/EditableRelationFilterChip.tsx

```diff
@@ -0,0 +1,42 @@
+import { useFieldMetadataItemByIdOrThrow } from '@/object-metadata/hooks/useFieldMetadataItemByIdOrThrow';
+import { type RecordFilter } from '@/object-record/record-filter/types/RecordFilter';
+import { SortOrFilterChip } from '@/views/components/SortOrFilterChip';
+import { useComputeRecordRelationFilterLabelValue } from '@/views/hooks/useComputeRecordRelationFilterLabelValue';
+import { useIcons } from 'twenty-ui/display';
+
+type EditableRelationFilterChipProps = {
+  recordFilter: RecordFilter;
+  onRemove: () => void;
+  onClick?: () => void;
+};
+
+export const EditableRelationFilterChip = ({
```

Comment:
Nice to create another component instead of having to many ternaries

Reply:
+ no choice since this is calling a heavy hook and hooks cannot be called conditionnaly